AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -244,6 +244,16 @@ internal class CardViewController: FormViewController {
 
         items.triggerInfoEvent(of: .selected, target: .dualBrandButton, brands: [selectedBrand])
     }
+
+    private func showCoBadgedCardsUI(for brands: [CardBrand]) {
+        let allowedCoBadgedCardTypes = ["cartebancaire", "bcmc", "dankort"]
```

Comment:
Maybe move this list as a constant to top of the class or in ?
Or ideally move this check to BinLookup because AFAIU in the future it will be part of API response